apps/loan/src/components/PageCrvUsdStaking/TransactionTracking/index.tsx

```diff
@@ -0,0 +1,22 @@
+import useStore from '@/store/useStore'
+
+import DepositTracking from './DepositTracking'
+import WithdrawTracking from './WithdrawTracking'
+
+type TransactionTrackingProps = {
+  className?: string
+  approved?: boolean
+}
+
+const TransactionTracking = ({ className }: TransactionTrackingProps) => {
+  const { stakingModule } = useStore((state) => state.scrvusd)
+
+  return (
+    <>
+      {stakingModule === 'deposit' && <DepositTracking className={className} />}
+      {stakingModule === 'withdraw' && <WithdrawTracking className={className} />}
+    </>
+  )
+}
```

Comment:
Another way to remove the logic from the UI is
```suggestion
const components = {deposit: DepositTracking, withdraw: WithdrawTracking}
const TransactionTracking = ({ className }: TransactionTrackingProps) => {
  const { stakingModule } = useStore((state) => state.scrvusd)
  const Component = components[stakingModule]
  return <Component className={className} />
}
```